api-catalog-services/src/main/java/org/zowe/apiml/apicatalog/health/ApiCatalogHealthIndicator.java

```diff
@@ -22,19 +24,40 @@
  */
 @Component
 @RequiredArgsConstructor
+@Slf4j
 public class ApiCatalogHealthIndicator extends AbstractHealthIndicator {
+    private static String AUTHENTICATION_SERVICE_ID = "apiml.authorizationService.zosmfServiceId";
+    private static String AUTHENTICATION_SERVICE_PROVIDER = "apiml.authorizationService.provider";
+    private static String ZOSMF = "zosmf";
 
     private final DiscoveryClient discoveryClient;
 
     @Override
     protected void doHealthCheck(Health.Builder builder) {
         String gatewayServiceId = CoreService.GATEWAY.getServiceId();
+        String authServiceId = ZOSMF;
+        String authServiceProvider = ZOSMF;
+        if (!this.discoveryClient.getInstances(gatewayServiceId).isEmpty()){
```

Comment:
How about replace this with:
`boolean gatewayUp = !this.discoveryClient.getInstances(gatewayServiceId).isEmpty();`
`if (gatewayUp ) {`
Because below we use the same check, in line 55.

Reply:
I merged the two logic checks.

Reply:
Thanks!